### ISSUE TITLE
Minion slabs should include text detailing a minion's special ability (if it has one)

### ISSUE THREAD

**Issue by Cruuk** [Tslat/Advent-Of-Ascension#199]
**Describe/name the existing feature this suggestion relates to**
Minion slabs.

**Describe the change or feature you think would be good to see**
We know certain minions have special abilities. Compeer slows enemies, Ender Carrier allows access to your ender chest, Blissard heals you on hit, etc...
But how is the the player supposed to know this without looking at the wiki, ~~like many other things in the mod.~~

Basically, like all the weapons in the mod, minion slabs would explain any abilities minions have.
Compeer's slab would have "slows enemies on hit", Ender Carrier's slab would have "right-click to access ender chest", etc.
Minions that have no special abilities like the penguin would have their slabs unchanged.
